huntsman/observatory.py

```diff
@@ -386,6 +386,74 @@ def take_evening_flats(self,
                 self.logger.debug('Waiting for dark-field image')
                 time.sleep(1)
 
+    def take_dark_fields(self,
+                         max_exptime=60.,
+                         camera_list=None,
+                         max_num_exposures=10,
+                         *args, **kwargs
+                         ):
+        """Take dark fields
+
+        Args:
+            max_exptime (float, optional): Maximum exposure time before stopping
+            camera_list (list, optional): List of cameras to use for darks
+            max_num_exposures (int, optional): Maximum number of darks to take
+            *args (TYPE): Description
+            **kwargs (TYPE): Description
+
+        """
+
+        image_dir = self.config['directories']['images']
+
+        dark_obs = self._create_dark_observation()
+        exp_times = {cam_name: [1. * u.second] for cam_name in camera_list}
```

Comment:
Looks like you're hard-coding 1 second exposures here for some reason. Shouldn't this be given as a arg to `take_dark_fields`?